### DISCUSSION TITLE
Which company's products are the motors used in iCub robots?

### DISCUSSION THREAD

**Discussion by xilihutu888** [robotology/community#536] [Q&A] [answered]
Which company's products are the motors used in iCub robots? What's the URL? Please help me, thanks.

**Comment by pattacini** [comment, marked as answer]
We do have Faulhaber DC motors for joints with low inertia (e.g. fingers, eyes) and custom-made MOOG BLDC motors for joints with bigger inertia (e.g. shoulders, legs).

cc @maggia80

**Reply by xilihutu888** [reply]
Hello, can you find the specific model of the motor on the official website? What's the URL? When will iCub accessories purchase website be launched? thanks.

**Reply by pattacini** [reply]
Hi @xilihutu888 

Find attached a detailed list of the motors mounted on our standard version of the iCub:

[iCub2.7-motors.xlsx](https://github.com/robotology/community/files/6886644/iCub2.7-motors.xlsx)

>What's the URL?

- https://www.faulhaber.com/en/home/
- https://www.moog.it/

>When will iCub accessories purchase website be launched?

There's no plan at the moment to publish the BOM of the robots.

**Reply by xilihutu888** [reply]
OK, thanks
